apps/web/lib/components/modal.tsx

```diff
@@ -40,12 +41,22 @@ export function Modal({
 					ref={refDiv}
 					className="absolute inset-0 flex items-center justify-center p-4"
 				>
-					<Dialog.Panel className={className}>
+					<Dialog.Panel className={`${className} relative`}>
 						{title && <Dialog.Title>{title}</Dialog.Title>}
 						{description && (
 							<Dialog.Description>{description}</Dialog.Description>
 						)}
-
+						<div
```

Comment:
@desperado1802 Keep checking responsive versions too.

<img width="486" alt="image" src="https://github.com/ever-co/ever-teams/assets/81486442/021d6547-e5a6-4407-9873-676f505a4711">

Reply:
Improved the position and respobsivness, also called the close function directly
![image](https://github.com/ever-co/ever-teams/assets/124465103/f4e034cc-012b-4748-ae16-e0e4a779023f)
![image](https://github.com/ever-co/ever-teams/assets/124465103/6c63a24e-5a9d-42c3-9de2-bb497925e3de)

Reply:
> Improved the position and respobsivness, also called the close function directly ![image](https://user-images.githubusercontent.com/124465103/255394311-a88734da-c9dc-46a8-9768-a8a8ba1c18c9.png)
> 
> ![image](https://user-images.githubusercontent.com/124465103/255394128-6c63a24e-5a9d-42c3-9de2-bb497925e3de.png)

@desperado1802 Top and Right padding should be same, not looking good in responsive device.

Reply:
![image](https://github.com/ever-co/ever-teams/assets/118497/290937aa-434d-4c85-92b7-d54f649e4e6b)

"Top and Right padding should be same" 100% agree with @badalkhatri0924 . In fact, it's one of the very important "features" (if not most important) of great engineers - attention to detail...